spec/system/candidate_interface/candidate_selecting_a_course_study_mode_spec.rb

```diff
@@ -23,7 +23,7 @@ def given_there_is_a_single_site_full_time_course
     @third_site = create(:site, provider: @provider)
 
     @single_site_course = create(
-      :course, :with_both_study_modes, :open_on_apply, provider: @provider
+      :course, :with_both_study_modes, :open_on_apply, provider: @provider, name: 'MS Painting'
```

Comment:
Doesn't the underlying problem still exist in the `:course` factory?
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/factories.rb#L155
Would it be better to fix this with some sort of FactoryBot sequence on the name?

Reply:
^ I like this suggestion

Reply:
@chubberlisk I misunderstood the issue, course names aren't unique so it's fair to fix the test in question.